ipynb/Animations (metadata curation).ipynb

```diff
@@ -0,0 +1,205 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "def load_mf(fp, index='#SampleID'):\n",
+    "    _df = pd.read_csv(fp, sep='\\t', keep_default_na=False, na_values=[], dtype=str)\n",
+    "    _df.set_index(index, inplace=True)\n",
+    "    return _df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the mapping files for the ICU, ITS and FMT from Qiita. Then combine them with the AG mapping file that you got from Justine."
```

Comment:
@jwdebelius, can you clarify which mapping file this is? Let me know if more things need to be staged on the ftp